pages/common/bmaptool.md

```diff
@@ -0,0 +1,15 @@
+# bmaptool
+
+> Create or Copy blockmaps intelligently (and therefore faster than `cp` or `dd`).
+
+- Create a blockmap into sdb1.
+
+`bmaptool create -o {{/dev/sdb1}}`
```

Comment:
I didn't use the tool itself, but as the example on its documentation `bmaptool create -o image.bmap image.raw`, I don't think the command here is correct (missing the image itself). Also I don't understand why one would want to create a blockmap *into* a single partition?

```
$ bmaptool create -o /dev/null
usage: bmaptool create [-h] [-o OUTPUT] [--no-checksum] image
bmaptool create: error: too few arguments
```